src/controllers/institution-controller.js

```diff
@@ -0,0 +1,149 @@
+import * as institutionService from '../services/institution-service.js'
+import { models } from '../models/init-models.js'
+import { Sequelize } from 'sequelize'
+import sequelizeConn from '../util/db.js'
+
+export async function fetchProjectInstitutions(req, res) {
+  try {
+    const institutions = await institutionService.fetchInstitutions(
+      req.params.projectId
+    )
+    res.status(200).json({ institutions })
+  } catch (e) {
+    console.error('Problem while getting project institutions (controller).', e)
+    res
+      .status(500)
+      .json({ message: 'Error while getting project institutions.' })
+  }
+}
+export async function addInstitutionToProject(req, res) {
+  const projectId = req.params.projectId
+  const name = req.body.name
+
+  if (name == null || name.replace(/\s/g, '').length == 0) {
```

Comment:
Can you replace `name.replace(/\s/g, '').length == 0` which `name.trim().length == 0` ?

But more importantly, you should `trim()` before you add it to the database just in case users add whitespaces to their institution - which they normally do.